bazelrio/dependencies/scripts/deps.bzl

```diff
@@ -3,11 +3,13 @@ load("@rules_python//python:pip.bzl", "pip_install")
 
 def setup_scripts_dependencies():
     maven_install(
+        name = "__bazelrio_maven_deps",
         artifacts = [
             "com.hierynomus:sshj:0.32.0",
             "me.tongfei:progressbar:0.9.2",
             "net.sourceforge.argparse4j:argparse4j:0.9.0",
             "org.slf4j:slf4j-nop:1.7.32",
+            "org.ejml:ejml-simple:0.38",
```

Comment:
I think this is a library dependency and not a script one.Perhaps this should be renamed to maven dependencies or something of the sort

Reply:
++ to having a different maven_install for wpimath